examples/magic_parameter_sweep.py

```diff
@@ -36,6 +36,7 @@ def gaussian_blur(
     """Apply a gaussian blur to ``layer``."""
     if layer:
         return skimage.filters.gaussian(layer.data, sigma=sigma, mode=mode)
+    return None
```

Comment:
should this type annotation be then `'Optional[napari.types.ImageData]'` then 🤔

Reply:
It will break napari. I'm not sure if fixing bug in `app_model.injections` is in the scope of this PR. 

I will create a separate PR and return to this after fixing a separate one.